python/test/unit/test_perf_warning.py

```diff
@@ -84,3 +94,37 @@ def ldst_vec(in_ptr0, in_ptr1, in_ptr2, in_ptr3, out_ptr0, XBLOCK: tl.constexpr)
     _, err = capfd.readouterr()
     assert ("remark: Warning: vectorization fails" in err), "expect vectorization failure remark"
     os.environ["MLIR_ENABLE_REMARK"] = "0"
+
+
+def test_remark_size_per_thread_equals_one(capfd, fresh_triton_cache):
```

Comment:
is there a more minimal test that would work here?

Reply:
I removed some tmp variables and removed two ops. Basically this is a relatively minimum 2d load and sum. It can still result in the blocked layout. 
I also added one more test that is simplified from this one, which is a 1d load and sum. But that could be somewhat different from this one, as it deliberately loads only one element from memory.